addons/interaction/XEH_postInit.sqf

```diff
@@ -108,3 +108,11 @@ GVAR(isOpeningDoor) = false;
 
 ["isNotSwimming", {!underwater (_this select 0)}] call EFUNC(common,addCanInteractWithCondition);
 ["isNotOnLadder", {getNumber (configFile >> "CfgMovesMaleSdr" >> "States" >> animationState (_this select 0) >> "ACE_isLadder") != 1}] call EFUNC(common,addCanInteractWithCondition);
+
+["ace_settingsInitialized", {
+    if (GVAR(disableNegativeRating)) then {
+        ACE_player addEventHandler ["HandleRating", {
```

Comment:
Are you sure? I am not sure how rating works with remote control, doesn't it get applied to the remote controlled unit? I guess we still wouldn't want to impact that if you are Zeusing or something though.